src/V3AstNodes.cpp

```diff
@@ -2326,6 +2326,58 @@ void AstCAwait::dump(std::ostream& str) const {
         sensesp()->dump(str);
     }
 }
+bool AstCMethodHard::isPure() {
+    if (!m_purity.isCached()) m_purity.setPurity(getPurity());
+    return m_purity.isPure();
+}
+const char* AstCMethodHard::broken() const {
+    static const char* impureMethods[]
+        = {"evaluate", "first", "last", "next", "pop_back", "pop_front", "prev", "set"};
```

Comment:
Most of `void` methods aren't in the map now. I will add them. We can't skip void methods, because we don't have `dtypep` field set in the constructor.